sormas-backend/src/main/resources/sql/sormas_schema.sql

```diff
@@ -9798,4 +9798,607 @@ ALTER TABLE weeklyreportentry_history ALTER COLUMN creationdate TYPE timestamp(3
 
 INSERT INTO schema_version (version_number, comment) VALUES (436, 'Set timestamp precision to milliseconds #7303');
 
+-- 2022-01-22 change by user #7323
+
+ALTER TABLE action ADD COLUMN change_user_id BIGINT,
+                   ADD CONSTRAINT fk_change_user_id
+                       FOREIGN KEY (change_user_id)
+                           REFERENCES users (id);
+
+ALTER TABLE action_history ADD COLUMN change_user_id BIGINT,
+                           ADD CONSTRAINT fk_change_user_id
```

Comment:
@JonasCir The history tables in general should not have foreign keys.

Reply:
fixed